src/kes.js

```
@@ -170,6 +170,8 @@ class Kes {
       const destPath = path.join(this.config.kesFolder, this.cf_template_name);
       console.log(`Template saved to ${destPath}`);
       return fs.writeFileSync(destPath, cf);
+    }).catch((e) => {
+      console.log(e);
```

I think we would also want to re-throw the error here or else it would be swallowed when `compileCF` is called elsewhere, like `uploadCF`

---

Makes sense, I added a throw